client/package.json

```diff
@@ -10,10 +10,12 @@
   },
   "dependencies": {
     "@vueuse/core": "^10.3.0",
+    "axios": "^1.4.0",
```

Comment:
I would suggest using fetch() instead of axios, since that is already included, and is more than enough for the kind of requests you are making. 
More here : https://blog.logrocket.com/axios-vs-fetch-best-http-requests/